test/runtests.jl

```diff
@@ -5,7 +5,7 @@ import MLJFlux
 using CategoricalArrays
 import Flux
 import Random.seed!
```

Comment:
Could you add `using Statistics` here? Looks like the travis test is failing because of this.

Reply:
Done. Hopefully that fixes it.